client/scss/layouts/_login.scss

```diff
@@ -1,5 +1,6 @@
 // overrides default nice padding defined in variables.scss
-$desktop-nice-padding: 100px;
+$custom-nice-padding: 100px;
```

Comment:
Worth making the nice padding thing a css custom property?

Reply:
I’d prefer not to embark on refactorings like this at this point in this PR – but let me know if you disagree.

Reply:
Agreed.